src/pages/gen1/[platform]/prev/build-a-backend/graphqlapi/mutate-data/index.mdx

```diff
@@ -56,10 +56,6 @@ import flutter3 from '/src/fragments/lib-v1/graphqlapi/flutter/mutate-data.mdx';
 
 In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client.
 
-Before you begin, you will need:
-
-- An [application connected to the API](/gen1/[platform]/prev/build-a-backend/graphqlapi/set-up-graphql-api/)
```

Comment:
I think we discussed removing this, but I don't remember why. Maybe we though it was a vCurrent reference from vPrev? It looks fine to me since it is to /prev from /prev. Let's leave this if there wasn't another reason to remove it.

Reply:
Same for the next 2 files